.cicd/test-package.run.sh

```diff
@@ -13,11 +13,13 @@ elif [[ $(yum --version 2>/dev/null) ]]; then # RHEL family packaging
     perform 'yum install -y /eos/*.rpm'
 elif [[ $(brew --version 2>/dev/null) ]]; then # homebrew packaging
     perform 'brew update'
+    perform 'brew --version'
     perform 'mkdir homebrew-eosio'
     perform 'git init homebrew-eosio'
     perform 'cp *.rb homebrew-eosio'
     perform "sed -i.bk -e 's/url \".*\"/url \"http:\/\/127.0.0.1:7800\"/' homebrew-eosio/*.rb"
     perform "pushd homebrew-eosio && git add *.rb && git commit -m 'test it!' && popd"
+    perform 'export HOMEBREW_NO_INSTALLED_DEPENDENTS_CHECK=false'
```

Comment:
This explains why installing it twice or running the fix links command works but it's hideous this is the default Homebrew behavior. I don't see a way to force this in the formulae itself.

Reply:
Can we try (rather than setting this export) `brew list --formula | xargs -I % sh -c 'brew unlink %; brew link %'` after `brew update` to see if its a problem with the existing installed formula?

Reply:
Test Package failed https://buildkite.com/EOSIO/eosio/builds/32239#3d89b1f3-3a2b-49f6-87f8-f07fc422c033 after trying `brew list --formula | xargs -I % sh -c 'brew unlink %; brew link %'`. Will keep setting this export as a fix since we can't prove it’s eosio, and the fix suggests its a problem with the existing installed packages.

Reply:
> the fix suggests its a problem with the existing installed packages.

I thought the point of this test is that this is a clean image with no existing installed formula. What `brew` packages are already installed?

Reply:
Updating my comments: the fix suggests some broken linkage of dependents or outdated dependents after installing, upgrading or reinstalling formulae.